client/messaging/message-list.tsx

```diff
@@ -79,16 +71,24 @@ export interface MessageListProps {
   loading?: boolean
   /** Whether this message list has more history available that could be requested. */
   hasMoreHistory?: boolean
+  /**
+   * A unique string value that's used by the infinite scroll list to decide when to refresh the
+   * list.
+   */
+  refreshToken?: string
```

Comment:
Same as the other one, I don't think this type (or the comment) is correct. I also don't think the comment really describes its purpose or how to generate it, "unique" implies it is unique to something (e.g. throughout the app). A better way to describe this is "a value that changes when the values the list is displaying change, e.g. if the list is now displaying a different chat channel"